pages/_app.js

```diff
@@ -1,10 +1,18 @@
+import { useState } from "react";
 import GlobalStyle from "../styles";
 
 export default function App({ Component, pageProps }) {
-	return (
-		<>
-			<GlobalStyle />
-			<Component {...pageProps} />
-		</>
-	);
+  //Hier States
```

Comment:
Use descriptive comments in English.
For states, use the standard variable naming convention i.e. productToBuy, setProductToBuy.